### ISSUE TITLE
Error with component with bin day is tomorrow.

### ISSUE THREAD

**Issue by mjefferys** [robbrad/UKBinCollectionData#581]
### Name of Council

North West Leicestershire

### Issue Information

My bin day is tomorrow and the component is failing to start as it looks like it's trying to parse tomorrow into a date time. I believe this is because the council website changes from date string to 'tomorrow' as per this screenshot. I assume it might also fail if the day is 'today' I'll confirm that tomorrow.

![image](https://github.com/robbrad/UKBinCollectionData/assets/11043414/d76ec795-4fc6-4662-99a0-bffb82ebaea4)

``` 2024-01-17 12:13:37.099 ERROR (MainThread) [custom_components.uk_bin_collection.sensor] Unexpected error fetching Leascroft data: time data 'Tomorrow' does not match format '%a %d %b'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 300, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/uk_bin_collection/sensor.py", line 133, in _async_update_data
    data = await self.hass.async_add_executor_job(self.ukbcd.run)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/uk_bin_collection/uk_bin_collection/collect_data.py", line 96, in run
    return self.client_code(
           ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/uk_bin_collection/uk_bin_collection/collect_data.py", line 115, in client_code
    return get_bin_data_class.template_method(address_url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/uk_bin_collection/uk_bin_collection/get_bin_data.py", line 78, in template_method
    bin_data_dict = self.parse_data(
                    ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/uk_bin_collection/uk_bin_collection/councils/NorthWestLeicestershire.py", line 80, in parse_data
    parsed_date = datetime.strptime(date, "%a %d %b")
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Tomorrow' does not match format '%a %d %b' 

```

### Verification

- [X] I searched for similar issues at https://github.com/robbrad/UKBinCollectionData/issues?q=is:issue and found no duplicates
- [x] I have checked my address/postcode/UPRN works on the council's website
- [X] I have provided a detailed explanation of the issue as well as steps to replicate the issue
- [X] I understand that this project is run by volunteer contributors therefore completion of this issue cannot be guaranteed

**Comment by mjefferys** [comment]
I've submitted a potential fix in #582

**Comment by mjefferys** [comment]
Fixed in #585
